man/patterns.Rd

```diff
@@ -12,14 +12,15 @@ and melt them into separate columns. See the \code{Efficient reshaping using
 data.tables} vignette linked below to learn more.
 }
 \usage{
-patterns(\dots, cols=character(0))
+patterns(\dots, cols=character(0), perl=FALSE)
 }
 \arguments{
   \item{\dots}{A set of regular expression patterns.}
   \item{cols}{A character vector of names to which each pattern is matched.}
+  \item{perl}{Passed to \code{grep}: \code{TRUE} to use PCRE, \code{FALSE} to use TRE.}
 }
 \seealso{
-  \code{\link{melt}},
+  \code{\link{melt}}, \code{\link{grep}}
```

Comment:
somewhat unusual -- why not use `\code{\link{grep}}` in the `\item{perl}` entry above? That is where I would expect to find the link as a reader.

Reply:
ok, I removed it